.github/workflows/main.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         service:
           - gps2tsdb
           - oada_upload
+          - cell-logger
```

Comment:
I think this has to be `cell_logger` ... I can't remember what something chokes on the `-`'s

Reply:
@facastiblancor probably should change this.. will need to rename all instances of `cell-logger` to `cell_logger` like https://github.com/OATS-Group/isoblue-avena/pull/63/commits/3abf9024df7fc3a55229cfd492f04fe54c7480f2